src/plugins/better-evaluation-view/better-evaluation-view.replacements

```diff
@@ -118,12 +118,12 @@ Add a value field to "undefined" labels
 
 *Find* => `undefined`
 ```js
-if ($options = $options || {}, isNaN($val) || !isFinite($val)) return { type: "undefined" }
+if (isNaN($val) || !isFinite($val)) return { type: "undefined" }
 ```
 
 *Replace* `undefined` with
 ```js
-if ($options = $options || {}, isNaN($val) || !isFinite($val)) return { type: "decimal", value: isNaN($val) ? "\\mathrm{NaN}" : $val === Infinity ? "\\infty" : $val === -Infinity ? "-\\infty" : "undefined"}
```

Comment:
Weird that the bundler/minifier put `$options = $options || {},` inside the if statement. Now that it's gone, good riddance I guess